sfge/src/test/java/com/salesforce/rules/unusedmethod/CollidingNamesTest.java

```diff
@@ -1,72 +0,0 @@
-package com.salesforce.rules.unusedmethod;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-
-/**
- * There are a bunch of weird edge cases that can happen when class names, variable names, and
- * method names collide. These tests validate those. They are complicated enough to justify being
- * their own file.
- */
-public class CollidingNamesTest extends BaseUnusedMethodTest {
```

Comment:
Are these tests not applicable anymore?

Reply:
These tests got superseded by other tests in other files, so there was no longer a need for this file.